tests/testbmpcodec.c

```diff
@@ -23,17 +23,22 @@ using namespace DllExports;
 #include <stdlib.h>
 #include "testhelpers.h"
 
-static WCHAR file[] = {'t', 'e', 'm', 'p', '.', 'b', 'm', 'p', 0};
+#if !defined(_WIN32)
+#include <unistd.h>
+#endif
+
+static const char *file = "temp/temp.bmp";
+static WCHAR wFile[] = {'t', 'e', 'm', 'p', '/', 't', 'e', 'm', 'p', '.', 'b', 'm', 'p', 0};
```

Comment:
the tests seems to fail, probably because you added it in a subfolder which fopen doesn't automatically create. I'd instead just keep it in the same folder and name it more uniquely, e.g. testbmpcodec.bmp